jax-rs.itests/src/main/java/test/JaxrsTest.java

```diff
@@ -875,6 +879,192 @@ public void testApplicationWithGenericExtensionAndApplicationSelect()
         assertEquals(404, response.getStatus());
     }
 
+    @Test
+    /**
+     * Propietary extension... not in the spec
+     * breaks application isolation
+     */
```

Comment:
clarify that the issue of isolation breaking only occurs if more than one application target the same http context.